### PR TITLE
Feature/msca 133

### PR DESCRIPTION
**Pull request by kels271828**
Interpretable solution format:
* Deleted `create_result_df` and `save_result_df`
* Added `create_fixed_df` and `create_random_df`

### CODE REVIEW

File: src/crosswalk/model.py [hunk 1]
```diff
@@ -529,44 +529,37 @@ def get_cov_names(self) -> List[str]:
...
-            folder (str): Path to the result folder.
-            filename (str): Name of the result. Default to `'result.csv'`.
+        Returns:
+            pd.DataFrame: Random effects result data frame.
         """
-        if not filename.endswith('.csv'):
-            filename += '.csv'
-        df = self.create_result_df()
-        df.to_csv(folder + '/' + filename, index=False)
+        random_df = pd.DataFrame({
```
Comment (zhengp0) [inline src/crosswalk/model.py:558]: @kels271828 Here should we differentiate behavior based `self.use_random_intercept` is true or false?

Comment (kels271828) [reply src/crosswalk/model.py:558]: Yes, I was wondering about that! Do we want to throw an exception, create a warning and return None, or just return None? Or return some other value?

Comment (zhengp0) [reply src/crosswalk/model.py:558]: From a user perspective it is nice to be reminded about the model setup. So we can print out a warning says that there isn't random intercept in the model and return the data frame. I think the current code will still work, but all the gamma or the random effects will be zero.

Comment (kels271828) [reply src/crosswalk/model.py]: Added a warning!

**Review by zhengp0 — APPROVED** [review]
Thanks @kels271828 ! I think it is good to go :)

**Pull request merged**